store/tikv/txn_committer.go

```diff
@@ -367,6 +377,89 @@ func (c *txnCommitter) Commit() error {
 	return nil
 }
 
+func (c *txnCommitter) prewriteBinlog() chan error {
+	if !c.shouldWriteBinlog() {
+		return nil
+	}
+	ch := make(chan error, 1)
+	go func() {
+		prewriteValue := c.txn.us.GetOption(kv.BinlogData)
+		var err error
+		if prewriteValue != nil {
```

Comment:
If the return value of c.shouldWriteBinlog() is ture, the value of `prewriteValue != nil` also is true.